source/conf.py

```diff
@@ -381,6 +381,10 @@
     'user': (f'{github_sponsors_url}/%s', '@'),  # noqa: WPS323
 }
 
+linkcheck_ignore = [
+    "http://localhost:8000",
```

Comment:
Let's generalize this a little bit more:
```suggestion
    "http://localhost:\d+",
```